file/file_test.go

```diff
@@ -63,7 +69,20 @@ func (*fakePutClient) CloseSend() error {
 	return nil
 }
 
+func generateFile(t *testing.T, data string) string {
+	// Create a temporary file
+	fileName := path.Join(t.TempDir(), "data")
+
+	// Write some text to the file
+	if err := os.WriteFile(fileName, []byte(data), 0644); err != nil {
+		t.Fatalf("unable to write temp file contents: %v", err)
+	}
+
+	return fileName
+}
+
 func TestPut(t *testing.T) {
+	fileName := generateFile(t, data)
```

Comment:
add 
`const data = "some really important data"`
above this line

and delete the global const

Reply:
done